components/core/src/ffi/ir_stream/encoding_methods.cpp

```diff
@@ -177,30 +165,11 @@ static void add_base_metadata_fields(
     metadata[cProtocol::Metadata::TimeZoneIdKey] = time_zone_id;
 }
 
-static bool append_constant_to_logtype(string_view constant, bool, string& logtype) {
-    size_t begin_pos = 0;
-    auto constant_len = constant.length();
-    for (size_t i = 0; i < constant_len; ++i) {
-        auto c = constant[i];
-        if (cVariablePlaceholderEscapeCharacter == c || is_variable_placeholder(c)) {
-            logtype.append(constant, begin_pos, i - begin_pos);
-            logtype += cVariablePlaceholderEscapeCharacter;
-            // NOTE: We don't need to append the character of interest
-            // immediately since the next constant copy operation will get it
-            begin_pos = i;
-        }
-    }
-    logtype.append(constant, begin_pos, constant_len - begin_pos);
+static bool append_constant_to_logtype(string_view constant, string& logtype) {
```

Comment:
Not a problem of this PR, but we should use empty namespace instead of `static` to be consistent

Reply:
Yeah, we'll resolve these when we apply clang-tidy suggestions to the codebase.